core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -565,12 +574,40 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
             height: 10rem;
         }
 
+        [data-dot-object="contentlet"]:empty {
+            width: 100%;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 4rem;
```

Comment:
Maybe it should be the same height as an empty container

Reply:
`:empty` will not cut it... because it will not catch if there is like `<p></p>` we will keep seeing the issue.

Reply:
We can set a `min-height` for contentlets, whether they are empty or not, and retain those styles when they are completely empty. How does that sound good? @fmontes @zJaaal

Reply:
That sounds good

Reply:
Done!